homeworks/svitlana-kovalenko/20-react-api/public/index.html

```
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```

Good job! 
But there are some issues, pls fix them:

1) Adding and deleting todos works not correctly. Context updates correctly, but UI is not corresponding and works strange... 
2) Long list is cropped and not scrolled
3) Duplicated todos error is not handled.
4) Pls add all appeals from HW19 to this homework